miqa/core/management/commands/populate_demo.py

```diff
@@ -0,0 +1,39 @@
+from allauth.account.admin import EmailAddress
+from django.contrib.auth.models import User
+import djclick as click
+from guardian.shortcuts import assign_perm
+
+from miqa.core.models import Project
+from miqa.core.tasks import import_data
+
+
+@click.command()
+def command():
```

Comment:
I think it's generally a good practice to treat CLI commands like this as an API surface and restrict the logic in them to handling CLI arguments and logging, then move the actual business logic stuff of resetting the server to a more centrally useful place. It keeps the CLI details out of the actual impactful code, and makes it easier later to do the reset from a scheduled celery task or from an API call or whatever we might want to do in the future.

In this case, this function would literally just call a different function in a different file and nothing else, so there's not a lot of value in doing that now.